src/peripherals/peripherals.md

```diff
@@ -0,0 +1,37 @@
+# Peripherals
+
+## What are Peripherals?
+
+Most Microcontrollers have more than just a CPU, RAM, or Flash Memory - they contain sections of silicon which are used for interacting with systems outside of the microcontroller. These components are collectively known as Peripherals, and are often used to manage communication with sensors, Bluetooth radios, screens, or touch pads.
+
+These peripherals are useful because they allow a developer to offload processing to them, avoiding having to handle everything in software. Similar to how a desktop developer would offload graphics processing to a video card, embedded developers can offload some tasks to peripherals allowing the CPU to spend it's time doing something else important, or doing nothing in order to save power.
+
+However, unlike graphics cards, which typically have a Software API like Vulkan, Metal, or OpenGL, peripherals are exposed to our CPU with a hardware interface, which is mapped to a chunk of the memory.
```

Comment:
My use of CPU here was generally that "MCU = CPU + RAM + Flash + Peripherals" or such. Though, either way I should clarify.

Reply:
I tend to agree, but there's a certain understanding of what goes into a CPU nowadays and that's a lot more than a plain ALU as it used to be 30 years ago so to avoid confusion I would not use the term CPU.